surveys/2023-annual-survey/questions.md

```diff
@@ -0,0 +1,769 @@
+# Survey questions
+
+Whether or not you use Rust Programming Language <https://rust-lang.org> today, we want to hear from you!
+
+The Rust Community Team has created this survey to help us gauge how we're doing, what can be improved, and how we can best engage with all of you as we move forward.
+
+This is your chance to have a say in the development priorities of Rust.
+
+Your answers will be anonymous. Any personal data you submit as a part of this survey will be handled in accordance with our policy as described in our [Frequently Asked Questions](https://github.com/rust-lang/surveys/blob/main/documents/Community-Survey-FAQ.md).
+
+We estimate it will take about 10-25 minutes to complete.
+
+## Rust Usage
+
+### Do you use Rust?
+
+Type: select one
+
+- Yes, I use Rust (for any purpose, even if you're just learning) [`NEXT`](#your-rust-experience)
+- No, I don't currently use Rust, but I have in the past [`NEXT`](#for-previous-rust-users)
+- No, I have never used Rust [`NEXT`](#for-non-rust-users)
+
+> **justification**
+>
+> Fundamental for cohort analysis
+
+## For previous Rust users
+
+### Why did you stop using Rust?
```

Comment:
I've often heard "compile times" or similar tooling problems raised, but there's no good options for that here.

Reply:
It is included in the "What would you prioritise" part. Do you think it should also be here? :)

Seems like more things could be duplicated here. FWIW, I have personally heard a lot of complaints about compile times, but not anyone claiming that it was the reason to stop using Rust.

Reply:
Ah, my field is game dev, where the iterative requirements are severe enough that it can actually be a show-stopper.

I think that the prioritization section + other option is fine here; it can be added next year if there's plenty of people talking about it.

Reply:
done